.github/workflows/react-editor.yml

```diff
@@ -0,0 +1,55 @@
+name: React Editor Application
```

Comment:
This will run a GitHub Action workflow that builds and lints the Editor Application.  I'll add running tests and coverage at another point in time.